src/components/NurseryWithdrawals/PlantingProgressMapDialog.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, Theme, Typography, useTheme } from '@mui/material';
+import { Button } from '@terraware/web-components';
+import { useCallback, useEffect, useMemo, useState } from 'react';
+import strings from 'src/strings';
+import { selectSubzoneSpeciesPopulations } from 'src/redux/features/tracking/sitePopulationSelector';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import FormattedNumber from 'src/components/common/FormattedNumber';
+import { makeStyles } from '@mui/styles';
+import Link from 'src/components/common/Link';
+import { requestUpdatePlantingCompleted } from 'src/redux/features/plantings/plantingsAsyncThunks';
+import { selectUpdatePlantingCompleted } from 'src/redux/features/plantings/plantingsSelectors';
+import useSnackbar from 'src/utils/useSnackbar';
+import { requestPlantingSites } from 'src/redux/features/tracking/trackingThunks';
+import { useOrganization } from 'src/providers';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  speciesList: {
+    margin: 0,
+    paddingLeft: theme.spacing(3),
+  },
+  withdrawalHistoryLink: {
+    textAlign: 'left',
+    fontSize: '16px',
+    fontWeight: 400,
+    marginTop: theme.spacing(3),
+  },
+}));
+
+type PlantingProgressMapDialogProps = {
+  id: number;
+  name: string;
+  plantingComplete: boolean;
+};
+
+export default function PlantingProgressMapDialog({
+  id,
+  name,
+  plantingComplete,
+}: PlantingProgressMapDialogProps): JSX.Element {
+  const theme = useTheme();
+  const classes = useStyles();
+  const dispatch = useAppDispatch();
+  const org = useOrganization();
+  const species = useAppSelector((state) => selectSubzoneSpeciesPopulations(state, id));
+  const totalPlants = useMemo(() => {
+    return Object.values(species).reduce((prev, curr) => prev + curr, 0);
+  }, [species]);
+  const [dispatching, setDispatching] = useState(false);
+  const [requestId, setRequestId] = useState<string>('');
+  const updateStatus = useAppSelector((state) => selectUpdatePlantingCompleted(state, requestId));
+  const snackbar = useSnackbar();
+
+  useEffect(() => {
+    if (updateStatus) {
+      if (updateStatus.status === 'error') {
+        snackbar.toastError(strings.GENERIC_ERROR);
+      }
+      // refresh planting site data to get new completed state for subzone
+      dispatch(requestPlantingSites(org.selectedOrganization.id));
+      setDispatching(false);
+    }
+  }, [updateStatus, dispatch, snackbar, org]);
```

Comment:
instead of org this should be org.selectedOrganization.id to avoid unwanted useEffect triggers